src/hb-aat-layout-trak-table.hh

```diff
@@ -64,21 +64,71 @@ struct TrackData
     return_trace (c->check_struct (this));
   }
 
-  protected:
+  public:
   HBUINT16		nTracks;	/* Number of separate tracks included in this table. */
   HBUINT16		nSizes;		/* Number of point sizes included in this table. */
   LOffsetTo<UnsizedArrayOf<Fixed> >
 			sizeTable;
-  TrackTableEntry	trackTable[VAR];/* Array[nSizes] of size values. */
+  UnsizedArrayOf<TrackTableEntry>	trackTable;/* Array[nSizes] of size values. */
 
   public:
   DEFINE_SIZE_ARRAY (8, trackTable);
 };
 
+#define HB_CORETEXT_DEFAULT_FONT_SIZE 12.f
```

Comment:
If ptem is <= 0 / not set, I *think* we should simply not apply trak.

Reply:
Definitely, make sense, its logic is copied from hb-coretext, perhaps we can change that also to share the logic sometime.

Reply:
Done